exercises/practice/reverse-string/.approaches/span/content.md

```diff
@@ -1,12 +1,20 @@
 # Span&lt;T&gt;
```

Comment:
Seems like some other change to an unrelated file is added in this PR. Expected or incorrect?

Reply:
Whoa. Missed this comment. It was unintended but correct